packages/ffe-searchable-dropdown-react/src/ListContainer.js

```diff
@@ -13,9 +13,13 @@ const ListContainer = ({
     maxRenderedDropdownElements,
 }) => {
     const itemsToRender =
-        maxRenderedDropdownElements > 0
-            ? dropdownList.slice(0, maxRenderedDropdownElements)
-            : dropdownList;
+        dropdownList.length > 0
```

Comment:
You have structured the lines in a nice way, but personly I am not a fan of nesting tenary.